app/components/header/navigation_component.rb

```diff
@@ -19,14 +20,130 @@ def all_resources
 
   private
 
-    def nav_link(link_text, link_path)
-      tag.li class: class_name(link_path) do
-        link_to_unless_current(link_text, link_path, class: "link--black link--no-underline") { tag.div(link_text) }
+    def sync_mode(mode)
+      mode == :mobile ? :desktop : :mobile
+    end
```

Comment:
this seems slightly counterintuitive at first glance

Reply:
maybe there's a way of expressing what `sync` means in this context

Reply:
yep I think I could phrase that better 👍